Vagrantfile

```diff
@@ -0,0 +1,30 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
+  config.vm.box = "ubuntu/focal64"
```

Comment:
Updated the base image from bionic (EOL in 2023) to focal (EOL in 2025). Note that the next LTS release of Ubuntu will come out shortly (in April 2022).